src/opensearch.py

```diff
@@ -104,6 +104,21 @@ def is_failed(self) -> bool:
 
         return service_failed("snap.opensearch.daemon.service")
 
+    def _set_env_variables(self):
```

Comment:
Why are these defined? these should be already defined from within the snap

Reply:
And add a `@override`